.github/workflows/kbs_build_and_push.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build and Push kbs Image
+
+on:
+  push:
+    branches:
+      - main
+      - kartikjoshi21/kbs-image
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Get Commit SHA
+      id: commit_sha
+      run: echo "::set-output name=sha::$(git rev-parse --short ${{ github.sha }})"
+
+    - name: Login to quay Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ secrets.repository_owner }}
+        password: ${{ secrets.GHCR_TOKEN }}
+
+    - name: Build Container Image
+      run: |
+        commit_sha=${{ steps.commit_sha.outputs.sha }}
+
+        DOCKER_BUILDKIT=1 docker build -t ghcr.io/kartikjoshi21/confidential-containers/kbs:kbs-${{ github.sha }} --build-arg KBS_FEATURES=coco-as-builtin,openssl,resource,opa . -f docker/Dockerfile --push
```

Comment:
Maybe we can push the image to coco's community repo https://github.com/orgs/confidential-containers/packages.

cc @fitzthum wdyt?

Then we need to think about the location of the images. Two ideas from my side.
1. The same repo as current kbs. The image name might be `ghcr.io/confidential-containers/key-broker-service:commit-${{ github.sha }}`
2. A new repo for staged images (We can think that AS/RVPS would also need on every merge to main). The image name might be `ghcr.io/confidential-containers/staged-images:kbs-commit-${{ github.sha }}`

Reply:
Maybe lets keep merge to main images as part of new repo to avoid confusion. What do you think ?